data/entities.yaml

```diff
@@ -378,7 +396,7 @@
 - name: plasma cutter
   display:
     attr: device
-    char: '+'
+    char: '*'
```

Comment:
This is also a flower, detonator and gear? `P` would make sense as a downward pointing plasma gun :thinking:

Reply:
They are all distinguished by color.  But P for plasma cutter sounds good.